aim_spacy/base_logger.py

```diff
@@ -75,13 +75,14 @@ def aim_log_step(info: Optional[Dict[str, Any]]):
                     if isinstance(other_scores, dict):
                         for score_name, loss_value in other_scores.items():
                             if not isinstance(loss_value, dict):
-                                aim_run.track(loss_value, name=loss_name, context={'type':f'other_scores_{score_name}'}, epoch=epoch, step=step)
+                                if not score_name.endswith('_desc'): 
```

Comment:
@osoblanco, please add comment describing why we need to skip these loss values.

Reply:
`_desc` stands for "Description" which is always a string.

Reply:
Comment added in code as well :)